.eslintrc.js

```diff
@@ -33,7 +33,7 @@ module.exports = {
     'import/no-cycle': [0, { ignoreExternal: true }],
     '@typescript-eslint/no-use-before-define': ['error', { functions: false, classes: false, variables: true }],
     '@typescript-eslint/no-namespace': ['error', { allowDeclarations: true }],
-    '@typescript-eslint/no-shadow': 'error',
+    '@typescript-eslint/no-shadow': ['error'],
```

Comment:
You should only need to provide an array if you're setting custom options. I imagine it'll work like this too, but 'error' is clear that we are using the default options

Reply:
I believe this is a due to a merge conflict, but I can change it back

Reply:
Bit cheeky but it looks like `'no-shadow': 'off',` has been duplicated at some point, would you mind cleaning that up too? Probs also due to a merge conflict.